tests/attr/neuron/test_neuron_conductance.py

```diff
@@ -138,7 +140,11 @@ def _conductance_input_test_assert(
                 )
 
     def _conductance_input_sum_test_assert(
-        self, model, target_layer, test_input, test_baseline=None
+        self,
+        model: Module,
+        target_layer: Module,
+        test_input: TensorOrTupleOfTensors,
+        test_baseline: Any = None,
     ):
```

Comment:
In this method, on lines 170 and 171, change these to the following to resolve errors:
```
                            torch.sum(neuron_vals[n]).item(),
                            attributions[n, i, j, k].item(),

```